_includes/page-type-info-box.html

```diff
@@ -0,0 +1,64 @@
+{% if page.permalink contains '/topics/' %}
+  {% assign pageType = 'topics' %}
+{% elsif page.permalink contains '/resources/' %}
+  {% assign pageType = 'resources' %}
+{% elsif page.permalink contains '/law-and-regs/' %}
+  {% assign pageType = 'law-and-regs' %}
+{% endif %}
+
+{% if page.tags %}
+  {% assign filters = page.tags | join: ";" | replace: " ", "-" | prepend: "?filters=" %}
+{% else %}
+  {% assign filters = "" %}
+{% endif %}
+
+{% if pageType %}
+
+<div
+  class="icon-box usa-summary-box border-0 padding-2 bg-primary-lighter"
+  role="region"
+  aria-labelledby="page-type-info-box-header"
+>
+  <div class="content-column">
+    <h3 class="usa-summary-box__heading" id="page-type-info-box-header">
+      {% case pageType %}
+        {% when 'topics' %}
+          {% include svg/library.html %}
+          <span>Featured Topic</span>
+        {% when 'resources' %}
+          {% include svg/folder.html %}
+          <span>Guidance & Resources</span>
+        {% when 'law-and-regs' %}
+          {% include svg/gavel.html %}
+          <span>Laws, Regulations, & Standards</span>
+      {% endcase %}
+    </h3>
+
+    <div class="usa-summary-box__text">
+      <div class="margin-1">
+        {% case pageType %}
+          {% when 'topics' %}
+            <p>This article is a basic introduction to this topic area. Read this page to get a high-level understanding.</p>
+          {% when 'resources' %}
+            <p>This article is a deeper dive into this topic area. Read this page to understand specific parameters on a topic.</p>
+          {% when 'law-and-regs' %}
+            <p>This article contains policy and legal regulations. Read this page to inform the creation of any legal policy, or to cite in a court of law.</p>
+        {% endcase %}
+
+        <ul class="no-bullets">
+          {% unless pageType == 'topics' %}
+          <li>For a higher level introduction, view <a class="usa-summary-box__link" href="/topics">Featured Topics</a></li>
+          {% endunless %}
+          {% unless pageType == 'resources' %}
+          <li>For a deeper dive, view <a class="usa-summary-box__link" href="/resources{{ filters }}">Guidance & Resource Materials</a></li>
+          {% endunless %}
+          {% unless pageType == 'law-and-regs' %}
+          <li>For any legal or policy applications, visit <a class="usa-summary-box__link" href="/law-and-regs">Laws, Regulations & Standards</a>
```

Comment:
was just creating a PR to remove the links and noticed you're missing an </li> here